Xamarin.Forms.Platform.iOS/iOSAppLinks.cs

```diff
@@ -107,13 +107,14 @@ static async Task<CSSearchableItemAttributeSet> GetAttributeSet(IAppLinkEntry de
 				Url = new NSUrl(deepLinkUri.AppLinkUri.ToString())
 			};
 
-			using (var uiimage = await deepLinkUri.Thumbnail.GetNativeImageAsync())
-			{
-				if (uiimage == null)
-					throw new InvalidOperationException("AppLinkEntry Thumbnail must be set to a valid source");
+			if (deepLinkUri.Thumbnail != null)
```

Comment:
I know the `using` block is sort of like a "single line", but I really think this would be more readable with braces.